README.md

```diff
@@ -63,32 +50,16 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Vet clinic database: create animals table <a name="about-project"></a>
 
-**[your_project__name]** is a...
+**Vet clinic database: create animals table** is a project that is part of the Microverse curriculum in SQL course. The goal of this project is to create a database for a vet clinic. The database will store information about animals, their owners, and the vets working at the clinic. The database will be created using PostgreSQL.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
- [ ] Kindly remove all the descriptive section on your README. Leaving it there does not make your README file look professional.